src/modules/messageEmbed.test.ts

```diff
@@ -0,0 +1,55 @@
+jest.useFakeTimers();
+import {strict as assert} from "assert";
+//import {extractFirstMessageLink} from "./messageEmbed";
+
+/*describe("modules/messageEmbed", () => {
```

Comment:
Not really a change request, just me wondering; why is this commented?

Reply:
I haven't been able to figure out how to setup the testing environment without it crashing. But since I had already written the tests, I decided to just leave it in there so it won't have to be redone in the future.